examples/http-circuit-breaker/http_circuit_breaker.bal

```diff
@@ -6,92 +6,93 @@ import ballerina/runtime;
 // The circuit breaker looks for errors across a rolling time window.
 // After the circuit is broken, it does not send requests to
 // the backend until the `resetTime`.
-endpoint http:Client backendClientEP {
-    url: "http://localhost:8080",
-    // Circuit breaker configuration options that control the
-    // behavior of the Ballerina circuit breaker
-    circuitBreaker: {
-        // Failure calculation window. This is how long Ballerina
-        // circuit breaker keeps the statistics for the operations.
-        rollingWindow: {
-
-            // Time period in milliseconds for which the failure threshold
-            // is calculated.
-            timeWindowMillis: 10000,
-
-            // The granularity at which the time window slides.
-            // This is measured in milliseconds.
-            // The `RollingWindow` is divided into buckets
-            //  and slides by these increments.
-            // For example, if this timeWindowMillis is set to
-            // 10000 milliseconds and bucketSizeMillis 2000.
-            // Then RollingWindow breaks into sub windows with
-            // 2-second buckets and stats will be collected with
-            // respect to the buckets. As time rolls a new bucket
-            // will be appended to the end of the window and the
-            // old bucket will be removed.
-            bucketSizeMillis: 2000,
-
-            // Minimum number of requests in a `RollingWindow`
-            // that will; trip the circuit.
-            requestVolumeThreshold: 0
+http:Client backendClientEP = new("http://localhost:8080", config = {
+        // Circuit breaker configuration options that control the
+        // behavior of the Ballerina circuit breaker
+        circuitBreaker: {
+            // Failure calculation window. This is how long Ballerina
+            // circuit breaker keeps the statistics for the operations.
+            rollingWindow: {
+
+                // Time period in milliseconds for which the failure threshold
+                // is calculated.
+                timeWindowMillis: 10000,
+
+                // The granularity at which the time window slides.
+                // This is measured in milliseconds.
+                // The `RollingWindow` is divided into buckets
+                //  and slides by these increments.
+                // For example, if this timeWindowMillis is set to
+                // 10000 milliseconds and bucketSizeMillis 2000.
+                // Then RollingWindow breaks into sub windows with
+                // 2-second buckets and stats will be collected with
```

Comment:
Let's say "stats are collected". 

It's better if we can avoid future tense. Normally the word "will" is indicative of future tense.